build/version.py

```diff
@@ -37,7 +37,7 @@ def _extractRevisionFromStdout(log, command, regex):
 
 def extractGitRevision(log):
 	return _extractRevisionFromStdout(
-		log, 'git describe --dirty', r'\S+?-(\S+)$'
+		log, 'git describe --always --dirty', r'\S+?-(\S+)$'
```

Comment:
Why is this necessary? We don't need it in openMSX or Catapult either.

Reply:
It causes warnings on git when repository is shallow cloned.

Reply:
I think openMSX and Catapult should have it too btw.

Reply:
I'd prefer to keep things in sync with openMSX and Catapult, if it's not strictly necessary, then I'd prefer to keep it the same. If it is, then we'd probably better change it in the other subprojects as well. Unfortunately, no one really proposed a way to keep the build system in sync between subprojects :(

Reply:
Can we apply it here then and then I make PRs for the other projects?

Reply:
Fine with me. I guess @mthuurne has no objections either.